test/e2e/tests/test_hosted_zone.py

```diff
@@ -109,6 +109,26 @@ def test_create_delete_private(self, route53_client, private_hosted_zone):
         route53_validator = Route53Validator(route53_client)
         route53_validator.assert_hosted_zone(zone_id)
 
+    def test_delegation_set(self, route53_client, public_hosted_zone):
```

Comment:
Seems like the tag key and value in the yaml file aren't being replaced, so you need this mark to override them.. 
```suggestion
    @pytest.mark.resource_data({'tag_key': 'key', 'tag_value': 'value'})
    def test_delegation_set(self, route53_client, public_hosted_zone):
```